src/modules/RootPart.rb

```diff
@@ -1240,6 +1240,25 @@ def MountFSTab(fstab, message)
         )
       end
 
+      efivars_path = "/sys/firmware/efi/efivars"
+      if ::File.exist?(efivars_path)
+        if MountPartition(efivars_path, "efivarfs", "efivarfs") == nil
+          AddMountedPartition(
+            { :type => "mount", :device => "efivarfs", :mntpt => efivars_path }
+          )
+        end
+      end
+
+      if SCR.Execute(
+          path(".target.mount"),
+          ["/run", Installation.destdir + "/run", Installation.mountlog],
```

Comment:
I'd prefer `File.join` to `+`

```suggestion
          ["/run", File.join(Installation.destdir, "run"), Installation.mountlog],
```